Assets/Scripts/Edges/EdgeGameObjectFactory.cs

```diff
@@ -50,5 +48,22 @@ void CreateNewEdgeGameObjectFor(Edge edge, IList<Vector3> edgePoints)
       lineRenderer.positionCount = positionCount;
       return (line, lineRenderer);
     }
+
+    private void GenerateCollider(GameObject line, Vector3 firstEdgePoint, Vector3 secondEdgePoint)
+    {
+      var collider = new GameObject(Constants.ColliderGameObjectName);
+      collider.transform.parent = line.transform;
+      var capsule = collider.AddComponent<CapsuleCollider>();
```

Comment:
what about cylinder collider instead of capsule collider?

Reply:
there is no cylinder collider component in unity